client/signup/steps/user/index.jsx

```diff
@@ -149,6 +150,7 @@ export class UserStep extends Component {
 				submitForm={ this.submitForm }
 				submitButtonText={ this.submitButtonText() }
 				suggestedUsername={ this.props.suggestedUsername }
+				isSocialSignupEnabled={ isEnabled( 'signup/social' ) && this.props.flowName === 'social' }
```

Comment:
I think it would be better to create a different step that implements the same component and then pass prop down, rather than rely on the flow name here.